docs/tutorial/setup-python.md

```diff
@@ -0,0 +1,28 @@
+=== "Cosma"
+
+    This tutorial is run on the [Cosma](https://cosma.readthedocs.io/en/latest/) supercomputer.
+    It should be straightforward to run on a different platform, the requirements are  `gcc`, `git` and `python3`. (for the later parts you also need `make`, `autotools`, `cmake` and `spack`).
```

Comment:
so any version of gcc and python3  ? Else give supported or min,max versions

Reply:
Python 3.7 or later is a requirement. For gcc or git I don't know if there's a minimum version required. I added the minimum versions supported by spack just in case.